### PR TITLE
Adding add_foreign function

### LINKED ISSUE #388: Add `add_foreign` to the schema builder
**Describe the feature as you'd like to see it**
We should have an `add_foreign` method on the schema builder. This should be a shortcut to the current way to add foreign keys. It should look like this:

```python
table.add_foreign("from_column.to_column.table")
```

The above would be the same as this:

```python
table.foreign("from_column").references("to_column").on("table")
```

### PR DESCRIPTION
**Pull request by alfonsocv12**
Closes #388 

This is the simples way that I could think of doing this, let me know what i need to change

### CODE REVIEW

File: src/masoniteorm/schema/Blueprint.py [hunk 1]
```diff
@@ -709,6 +709,21 @@ def primary(self, column=None):
...
         return self
 
+    def add_foreign(self, columns: string):
+        """Creates the foreign spliting the foreign name, reference column, and
+        reference table.
+
+        Arguments:
+            columns {string} -- The name of the from_column . to_column . table
+        """
+        columns_array: list = columns.split(".")
```
Comment (josephmancuso) [inline src/masoniteorm/schema/Blueprint.py]: I found using unpacking is more clear.

Changing this line to 
```python
        from_column, to_column, table = columns.split(".")
```

makes things a little more clear on what is going on

**Review by josephmancuso — CHANGES_REQUESTED** [review]
Perfect! pretty simple ticket just a small change I would make to make the code clearer

**Comment by josephmancuso** [comment]
Also should add a test in one of the grammar test classes like `tests/mysql/schema/test_mysql_schema_builder.py`

File: src/masoniteorm/schema/Blueprint.py (hunk 1 above)
Comment (alfonsocv12) [reply src/masoniteorm/schema/Blueprint.py]: And remove the validation down on leve it ?

**Review by josephmancuso — APPROVED** [review]
Looks great! I'l manually test this when I get time later but looks good.

**Review by josephmancuso — APPROVED** [review]

**Pull request merged**